proto/internal/temporal/server/api/persistence/v1/task_queues.proto

```diff
@@ -71,11 +72,35 @@ message CompatibleVersionSet {
     temporal.server.api.clock.v1.HybridLogicalClock became_default_timestamp = 4;
 }
 
+message AssignmentRule {
+    temporal.api.taskqueue.v1.BuildIdAssignmentRule rule = 1;
+    // (-- api-linter: core::0142::time-field-type=disabled
+    //     aip.dev/not-precedent: Using HLC instead of wall clock. --)
+    temporal.server.api.clock.v1.HybridLogicalClock create_timestamp = 2;
+    // (-- api-linter: core::0142::time-field-type=disabled
+    //     aip.dev/not-precedent: Using HLC instead of wall clock. --)
+    temporal.server.api.clock.v1.HybridLogicalClock delete_timestamp = 3;
```

Comment:
just comment that if `delete_timestamp` is present that means the rule should be treated as deleted